python/ctsm/modify_fsurdat/modify_fsurdat.py

```diff
@@ -21,7 +24,8 @@ class ModifyFsurdat:
 
     def __init__(self, fsurdat_in, lon_1, lon_2, lat_1, lat_2, landmask_file):
 
-        print("Opening fsurdat_in file to be modified: " + fsurdat_in)
+        logger.info(
+            'Opening fsurdat_in file to be modified: ' + fsurdat_in)
```

Comment:
@billsacks I realize that replacing the two `print` statements w `logger.info` statements may be overkill, but I wanted to make sure I had the basic logging infrastructure in this tool.